ReClass.NET/DataExchange/ReClass/ReClassNetFile.Constants.cs

```diff
@@ -1,9 +1,11 @@
-﻿namespace ReClassNET.DataExchange.ReClass
+namespace ReClassNET.DataExchange.ReClass
```

Comment:
What are these changes here and in `ReClass.NET_Launcher/Program.cs`? BOM or something?